src/sdp/DefaultSDP.ts

```diff
@@ -218,41 +218,95 @@ export default class DefaultSDP implements SDP {
       )
     );
     const srcLines: string[] = this.lines();
-    const dstLines: string[] = [];
+    const fmtpAttributes = DefaultSDP.findOpusFmtpAttributes(srcLines);
+    const dstLines = DefaultSDP.updateOpusFmtpAttributes(srcLines, fmtpAttributes, [
+      `maxaveragebitrate=${maxAverageBitrate}`,
+    ]);
+    return DefaultSDP.linesToSDP(dstLines);
+  }
+
+  withStereoAudio(): DefaultSDP {
+    const srcLines: string[] = this.lines();
+    const fmtpAttributes = DefaultSDP.findOpusFmtpAttributes(srcLines);
+    const dstLines = DefaultSDP.updateOpusFmtpAttributes(srcLines, fmtpAttributes, [
+      'stereo=1',
+      'sprop-stereo=1',
+    ]);
+    return DefaultSDP.linesToSDP(dstLines);
+  }
+
+  // Here we loop through each line in the SDP
+  // and construct an array containing the fmtp
+  // attribute for all the audio m lines that use
+  // the opus codec. If it doesn't use opus codec
+  // we add null to the array
+  static findOpusFmtpAttributes(sdpLines: string[]): string[] | null {
```

Comment:
nit: Could we use the same format use everywhere else?
/**
*
**/

Reply:
Updated